apis/r/DESCRIPTION

```diff
@@ -54,7 +54,10 @@ Imports:
 LinkingTo:
     Rcpp,
     RcppSpdlog
-Additional_repositories: https://ghrr.github.io/drat
+Additional_repositories:
+    https://ghrr.github.io/drat,
+    https://mojaveazure.r-universe.dev/bin/linux/jammy/4.3,
+    https://mojaveazure.r-universe.dev
 Remotes: bioc::release/SingleCellExperiment
```

Comment:
I missed this earlier and would be game for removing it.  `remotes` is not a batteries-include base R construct understood or used by CRAN.  There is only one gold standard, and that is CRAN.

Reply:
The `Remotes` entry is for CI as the CI will not install SingleCellExperiment without it (it didn't in [the SCE-outgest PR](https://github.com/single-cell-data/TileDB-SOMA/pull/1279)). I'd be happy to remove it, but would need help adjusting the CI to detect Bioc dependencies

Reply:
There are different routes to get from point A to point B.  CI, as set up, defaults to `remotes::install_deps()` because that generally works.  But we are not limited to it.  We can shimmy in a step to install, say, `BiocManager` and then `SingleCellExperiment`.  I can take a poke at it.

Reply:
Per a (really quick) test doing 

```r
install.packages("BiocManager")
BiocManager::install("SingleCellExperiment",  update=FALSE)
```

works and leaves other packages alone.  

So we could add an entry

```yaml
      - name: Install BioConductor package SingleCellExperiment
        run: cd apis/r && install_r BiocManager && Rscript -e 'BiocManager::install("SingleCellExperiment", update=FALSE)'
```        

which should work  -- but I need to add the Bioc repo to r-ci.sh for it to work on macOS.  I will prepare a quick PR.

Reply:
PR #1589 should add the ability to achieve this.

Reply:
With #1589 and 6a50e50, the `Remotes` field is gone and SingleCellExperiment is fetched through the CI rather than via `remotes::install_deps()`

Reply:
Yay!

(The one thing I am confused about a little is how it worked all along for `SummarizedExperiment` and `S4Vectors`.)

Reply:
SummarizedExperiment and S4Vectors are dependencies for SingleCellExperiment; with the `Remotes` field, `remotes::install_deps()` knew to look on Bioconductor for SingleCellExperiment and all of its dependencies, bringing along SummarizedExperiment and S4Vectors

Now, BiocManager is looking on Bionconductor for SingleCellExperiment and all of its depdencies

Reply:
Thanks I suspected just that.  In other words: it used to work with the remotes: field, we removed the remotes: field, and S4Vectors and SummarizedExperiment come in as dependencies of SingleCellExperiment.  

Do you know if there is way to rely on the remotes package but have it use BiocManager?  Ie remotes package, no remotes: field?

Reply:
AFAIK no; remotes relies on repositories and the `Remotes` field for its information. There was a period of time where adding a `biocViews` field to DESCRIPTION (even an empty one) would cause remotes to search Bioconductor, but that was unreliable and could cause issues where remotes would search exclusively on Bioconductor instead of CRAN + Bioconductor. Basically, no one has created a reliable way to declare that a package has Bioconductor dependencies and to search both CRAN and Bioconductor for them

Reply:
r2u does a good job and can help with the needed BioConductor packages _if_ it knows about all the BioConductor packages.  In other it does not know either when/how to call BiocManager.  It's a tricky issue.